src/main/java/racingcar/view/OutputView.java

```diff
@@ -0,0 +1,38 @@
+package racingcar.view;
+
+import java.util.List;
+import racingcar.model.RacingCars;
+
+public class OutputView {
+
+    private static final String RESULT_MESSAGE = "실행 결과";
+    private static final String RESULT_SEPARATOR = " : ";
+    private static final String MOVE_MARKER = "-";
+    private static final String COMMA = ",";
+    private static final String FINAL_WINNER = "최종 우승자 : ";
+    private static final String NO_WINNER = "우승자가 없습니다.";
+
+    public void printRaceStart() {
+        System.out.println(RESULT_MESSAGE);
+    }
+
+    public void printRaceStatus(RacingCars racingCars) {
+        racingCars.getCars().forEach(car -> printCarStatus(car.getName(), car.getPosition()));
+        System.out.println();
+    }
+
+    private void printCarStatus(String name, int position) {
+        String progress = MOVE_MARKER.repeat(position);
+        System.out.println(name + RESULT_SEPARATOR + progress);
+    }
```

Comment:
이렇게 쓰면 더 가독성에 좋을 것 같아요!! 

```suggestion
    public void printRaceStatus(RacingCars racingCars) {
        racingCars.getCars().forEach(this:: printCarStatus);
        System.out.println();
    }
    
    private void printCarStatus(Car car) {

    }    
    private void printCarStatus(Car car) {
        String progress = MOVE_MARKER.repeat(car.getPosition());
        System.out.println(car.getName() + RESULT_SEPARATOR + progress);
    }
```